docs/examples/exchanges/fanout-exchange-with-args.yaml

```
@@ -0,0 +1,23 @@
+# More on fanout exchange and other exchange types, see: https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchanges.
+---
+apiVersion: rabbitmq.com/v1beta1
+kind: Exchange
+metadata:
+  name: fanout-alt-exchange # name of the object in kubernetes
+spec:
+  name: fanout-alt-exchange # name of the exchange
+  vhost: "/test-vhost" # default to '/' if not provided
+  type: fanout # default to 'direct' if not provided
+  autoDelete: false
+  durable: true
+  arguments: # additional to provide towards the exchange, may be left empty.
+    - alternate-exchange: direct-exchange

```

This is an object, not an array.

```suggestion
  arguments: # additional to provide towards the exchange, may be left empty.
    alternate-exchange: direct-exchange
```

---

You are completely right! I didn't get around to fixing this yet as I wanted to go over the examples one more time. But accepted the change